airflow-core/src/airflow/api/common/trigger_dag.py

```diff
@@ -63,8 +63,8 @@ def _trigger_dag(
     :return: list of triggered dags
     """
     dag = dag_bag.get_dag(dag_id, session=session)  # prefetch dag if it is stored serialized
-
-    if dag is None or dag_id not in dag_bag.dags:
+    dag_info = dag_id, None  # Using None since we don't have bundle_version information
+    if dag is None or dag_info not in dag_bag.dags:
```

Comment:
the RHS of the OR condition will always evaluate to false pretty sure

e.g.
```
('a', None) in {'a': None}
Out[18]: False
```

Reply:
`not in dag_bag.dags` will return the list of the keys, something like: [('a', None),('b', None)] etc., not a dictionary

Reply:
Example data:
```
{('latest_only', None): <DAG: latest_only>, ('example_local_kubernetes_executor', None): <DAG: example_local_kubernetes_executor>}
```

Reply:
but that's weird cus, if it's dagbag.dags, it seems the type is `dict[str, DAG]`.
<img width="434" alt="image" src="https://github.com/user-attachments/assets/ebd03106-8ed2-4219-9560-b5c6c3d575fb" />
is this right?

Reply:
The type was changed in this PR